src/index.ts

```diff
@@ -84,11 +88,11 @@ const main = async () => {
   // create an oauth-client
   const dtClient = await createOAuthClient(oauthClient, oauthClientSecret, dtEnvironment, scopes);
 
-  console.error("Starting Dynatrace MCP Server...");
+  console.error(`Starting Dynatrace MCP Server v${VERSION}...`);
```

Comment:
This was pre-existing, but only caught my eye now: `console.error` is an odd choice. Is this intentional?

Reply:
Yes. With `STDIO` based mcp servers, we need to reserve Standard Input / Output for the MCP Protocol as much as possible. I've had issues in the past, so I chose the path of least resistance, which is console.error.

We can improve on this though in another ticket, and see how much we can do with console.log without breaking the protocol.